src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -258,7 +258,9 @@ public async Task<ChainedHeader> BlockMinedAsync(Block block)
                     if (fullValidationRequired)
                     {
                         ConnectBlocksResult fullValidationResult = await this.FullyValidateLockedAsync(validationContext.ChainedHeaderToValidate).ConfigureAwait(false);
-                        if (!fullValidationResult.Succeeded)
+                        if (fullValidationResult.Succeeded)
+                            this.chainedHeaderTree.ClaimPeerTipForMiner(validationContext.ChainedHeaderToValidate.HashBlock);
```

Comment:
If we have an else with braces then I prefer them also on the if if you dont mind.

Reply:
Done 👍